.github/workflows/ci_tests.yml

```diff
@@ -117,7 +117,7 @@ jobs:
       - name: Build Release
```

Comment:
Can we do parallel jobs ? e.g. `Build Release` || `Build Debug`

Reply:
The only way to run these in parallel is to separate them into multiple jobs. This would bring speedup but it would make number of checks double.